torch/_inductor/select_algorithm.py

```diff
@@ -498,7 +498,7 @@ def generate(
         ), TritonTemplateKernel(
             kernel_name=kernel_name,
             output_node=fake_out,
-            use_jit=True,
+            use_jit=False,
```

Comment:
After this, is there any remaining cases that set `use_jit` to True? If not, we can probably remove this argument